web/.env.example

```
@@ -19,4 +19,7 @@ CLICKUP_WORKSPACE_ID=
 
 # Your ClickUp user ID (numeric)
 # Find it via API: GET https://api.clickup.com/api/v2/user
-CLICKUP_USER_ID=
\ No newline at end of file
+CLICKUP_USER_ID=
+
+# Google Analytics 4
+PUBLIC_GA_MEASUREMENT_ID=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)